src/Sdk.php

```diff
@@ -76,7 +76,7 @@ public function __construct(array $args = [])
         }
     }
 
-    public function __call($name, array $args = [])
+    public function __call($method, $args)
```

Comment:
- Changing `$name` to `$method` doesn't work at all.
- I don't believe there is a need to remove the `array` typehint.
- But yes, the ` = []` should be removed.